lib/elixir/pages/Typespecs.md

```diff
@@ -170,7 +170,7 @@ Guards can be used to restrict type variables given as arguments to the function
 
 If you want to specify more than one variable, you separate them by a comma.
 
-    @spec function(arg1, arg2) :: [arg1, arg2] when arg1: atom, arg2: integer
+    @spec function(arg1, arg2) :: [arg1 | arg2] when arg1: atom, arg2: integer
```

Comment:
I'm not sure if this is valid either. `{arg1, arg2}` is definitely more come, so please change to that.